src/module/config.mjs

```diff
@@ -11,6 +11,17 @@ export const DRAW_STEEL = {};
  */
 DRAW_STEEL.characteristics = ["mgt", "agl", "rea", "inu", "prs"];
 
+/**
+ * @enum {string}
+ */
+DRAW_STEEL.sizes = {
+  T: "DRAW_STEEL.Actor.base.sizes.T",
+  S: "DRAW_STEEL.Actor.base.sizes.S",
+  M: "DRAW_STEEL.Actor.base.sizes.M",
+  L: "DRAW_STEEL.Actor.base.sizes.L"
+};
```

Comment:
If over the years I have learned anything, it is to avoid a config object that is just a record of strings. They always end up having to be migrated to objects, so let's just do this immediately.
```suggestion
DRAW_STEEL.sizes = {
  T: {
    label: "DRAW_STEEL.Actor.base.sizes.T"
  },
  S: {
    label: "DRAW_STEEL.Actor.base.sizes.S"
  },
  M: {
    label: "DRAW_STEEL.Actor.base.sizes.M"
  },
  L: {
    label: "DRAW_STEEL.Actor.base.sizes.L"
  }
};
```